.pre-commit-config.yaml

```diff
@@ -26,15 +26,16 @@ repos:
           files: ^.+$(?<!\.jpg)(?<!\.png)(?<!\.ipynb)(?<!\.bmp)(?<!\.tif)
           args: [ '--maxkb=500' ]
     - repo: https://github.com/psf/black
-      rev: 22.12.0
+      rev: 22.3.0
```

Comment:
I don't know what motivated this downgrade - probably not needed

Reply:
Bumped in [1c1657e](https://github.com/stfc/pre-commit-hooks/pull/32/commits/1c1657ed7c77b8e306fb0866104c0b6e790fc235).